### DISCUSSION TITLE
Docker image or other "all in one" installs?

### DISCUSSION THREAD

**Discussion by epugh** [Art-of-WiFi/UniFi-API-client#184] [General]
Hi..   ARe there any Docker or other "all in one" installs out there to make it easy for a non PHP person to get started?   Or any other blogs about how folks have used this?   I'd like to automate block and unblock of my teen's desktop according to specific times ;-)

**Comment by malle-pietje** [comment]
It doesn’t really make sense to spend time on creating a Docker distribution because the API is typically used by developers. That being said you could deploy the API browser using Docker, it comes with the API client bundled:
https://github.com/Art-of-WiFi/UniFi-API-browser/wiki/Docker-Hosting
